sys-fs/dwarfs/dwarfs-0.12.3-r1.ebuild

```diff
@@ -60,12 +60,8 @@ RESTRICT="!test? ( test )"
 CHECKREQS_DISK_BUILD="500M"
 CMAKE_WARN_UNUSED_CLI=0
 
-src_prepare(){
-	cmake_src_prepare
-	sed "s/DESTINATION lib/DESTINATION $(get_libdir)/" -i cmake/libdwarfs.cmake || die
```

Comment:
Thanks for following up on this. I guess the commit message can explain that it used to be necessary, but isn't *anymore*, and reference https://github.com/mhx/dwarfs/commit/ca69d2b141356b4cbb8d16d2607bd002faaf121a ?

Reply:
Done, wasn't sure how much detail was needed :)